src/proxy/query.go

```diff
@@ -218,13 +218,13 @@ func (spanner *Spanner) ComQuery(session *driver.Session, query string, callback
 			switch snode.From[0].(type) {
 			case *sqlparser.AliasedTableExpr:
 				aliasTableExpr := snode.From[0].(*sqlparser.AliasedTableExpr)
-				tb := aliasTableExpr.Expr.(sqlparser.TableName)
-				table := tb.Name.String()
-				if table == "dual" {
+				tb, ok := aliasTableExpr.Expr.(sqlparser.TableName)
```

Comment:
If table == "dual" and not ok is fasle, can we reproduce the  problem in unit test ?

Reply:
add